docs/snippets/react/page-story-slots.js.mdx

```diff
@@ -7,17 +7,16 @@ import Page from './Page';
 
 export default {
   component: Page,
-  title: 'Page',
 };
 
-const Template = (args) => (
-  <Page {...args}>
-    <footer>{args.footer}</footer>
-  </Page>
-);
-
-export const CustomFooter = Template.bind({});
-CustomFooter.args = {
-  footer: 'Built with Storybook',
+export const CustomFooter = {
+  args: {
+    footer: 'Built with Storybook',
+  },
+  render: (args) => (
+    <Page {...args}>
+      <footer>{args.footer}</footer>
+    </Page>
+  ),
```

Comment:
I don't quite understand this story either -- could you help me out? 😄  Does the page need those args? Does the page have a footer that this story is overriding?

Reply:
@darleendenno this snippet is tied to this [section](https://storybook.js.org/docs/react/writing-stories/args#args-can-modify-any-aspect-of-your-component) of the documentation. This was contributed by a community member, which in the use case he/she was trying to add a custom footer to his/her page.

Reply:
Thank you!! That's very helpful. This looks good 👍